backend/src/main/java/codezap/member/controller/SpringDocMemberController.java

```diff
@@ -26,321 +25,65 @@ public interface SpringDocMemberController {
-                                      "status": 400,
-                                      "detail": "비밀번호가 입력되지 않았습니다.",
-                                      "instance": "/login"
-                                    }
-                                    """
-                            ),
-                            @ExampleObject(name = "비밀번호 형식 오류", value = """
-                                    {
-                                      "type": "about:blank",
-                                      "title": "BAD_REQUEST",
-                                      "status": 400,
-                                      "detail": "영어와 숫자를 포함해야합니다.",
-                                      "instance": "/login"
-                                    }
-                                    """
-                            ),
-                            @ExampleObject(name = "비밀번호 글자수 오류", value = """
-                                    {
-                                      "type": "about:blank",
-                                      "title": "BAD_REQUEST",
-                                      "status": 400,
-                                      "detail": "비밀번호는 8~16자 사이로 입력해주세요.",
-                                      "instance": "/login"
-                                    }
-                                    """
-                            )
-                    }
-            )
-    )
-    @ApiResponse(
-            responseCode = "401",
-            description = "요청 형식 오류",
-            content = @Content(
-                    schema = @Schema(implementation = ProblemDetailSchema.class),
-                    examples = {
-                            @ExampleObject(name = "이메일 불일치", value = """
-                                    {
-                                      "type": "about:blank",
-                                      "title": "UNAUTHORIZED",
-                                      "status": 401,
-                                      "detail": "이메일이 입력되지 않았습니다.",
-                                      "instance": "/login"
-                                    }
-                                    """
-                            ),
-                            @ExampleObject(name = "비밀번호 불일치", value = """
-                                    {
-                                      "type": "about:blank",
-                                      "title": "UNAUTHORIZED",
-                                      "status": 401,
-                                      "detail": "비밀번호가 입력되지 않았습니다.",
-                                      "instance": "/login"
-                                    }
-                                    """
-                            ),
-                    }
-            )
-    )
+    @ApiResponse(responseCode = "200", description = "로그인 성공",
+            headers = @Header(name = "Set-Cookie", description = "base64(${email}:${password}); path=\"/\"; HttpOnly;"
+                    + " Secure;"))
```

Comment:
```suggestion
    @ApiResponse(responseCode = "200", description = "로그인 성공",
            headers = @Header(name = "Set-Cookie", description = "base64(${email}:${password}); path=\"/\"; HttpOnly; Secure;"), 
            content = {@Content(schema = @Schema(implementation = LoginResponse.class))})
```

⚡ 성공 케이스일 때 어떤 response가 반환되는지 명시해주세요

Reply:
따로 커스텀하지 않는다면 자동으로 인식된다고 알고 있어서 `@Schema` 를 명시적으로 추가하지 않았슴다!